aws-cs-fargate/Infra/FargateStack.cs

```diff
@@ -122,8 +122,8 @@ public FargateStack()
 
         // Create a private ECR registry and build and publish our app's container image to it.
         var appRepo = new Ecr.Repository("app-repo");
-        var appRepoCredentials = Ecr.GetCredentials
-            .Invoke(new Ecr.GetCredentialsInvokeArgs { RegistryId = appRepo.RegistryId })
+        var appRepoCredentials = Ecr.GetAuthorizationToken
+            .Invoke(new Ecr.GetAuthorizationTokenInvokeArgs { RegistryId = appRepo.RegistryId })
             .Apply(credentials =>
             {
                 var data = Convert.FromBase64String(credentials.AuthorizationToken);
```

Comment:
I think this needs to be updated as well. The `GetAuthorizationToken` result has the username and password available as  fields, you don't need to parse the output anymore.